phpBB/styles/prosilver/theme/common.css

```diff
@@ -603,6 +604,53 @@ a.header-avatar span:after {
 	display: none !important;
 }
 
+/* Smiley Dropdown---------------------------------------- */
+.smiley-box {
+	float: right;
+	width: 100%;
+}
+
+.smiley-box .smiley-image {
+	margin: 3px;
+}
+
+.dropdown .smiley-content {
+	width: 200px;
+}
+
+.smiley-box .smiley-list-item {
+	border: none;
+	display: inline-block;
+}
+
+.smiley-list-item .smiley-add {
+	padding: 0;
+}
+
+.button .dropdown-left .dropdown {
+	right: -12px;
+}
+
+.button .dropdown-right .dropdown {
+	left: -12px;
+}
+
```

Comment:
yes, I'll modify the footer.

To center thepointer I just only used following classes
.button .dropdown-left .dropdown {
	right: -12px;
}

.button .dropdown-right .dropdown {
	left: -12px;
}
.dropdown-right,.dropdown-left are existing classes and there is style to center the pointer inside that. They have used absolute value to position the pointer. So I just added left and right pixels to get center the pointer.